controllers/suite_test.go

```diff
@@ -20,18 +20,17 @@ import (
 	"path/filepath"
 	"testing"
 
+	// +kubebuilder:scaffold:imports
+	. "github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	capiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 
-	infrastructurev1beta1 "sigs.k8s.io/cluster-api-provider-ibmcloud/api/v1beta1"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
```

Comment:
wondering if there a way to allow dot(.) imports in the end, if not - wondering if we can ignore warning/error message for the dot import(prefer having them in the end) easy to see what imports are dot in different section.

Reply:
Added exception and fixed.